### PR TITLE
Add vestigial objects one-pager

### PR DESCRIPTION
**Pull request by garath**
This adds a one-pager for the "Vestigial Objects" effort planned as part of dotnet/arcade#8814.

Closes dotnet/arcade#10374.

### CODE REVIEW

File: Documentation/Project-Docs/vestigial-objects-onepager.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# "Vestigal objects" one-pager
```
Comment (MattGal) [inline Documentation/Project-Docs/vestigial-objects-onepager.md:1]: Nit: Spelling (Vestigial).  Feel free to use some other term for this too, of course.

File: Documentation/Project-Docs/vestigial-objects-onepager.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+The Resource Graph, Event Logs, and Resource Logs are stores of audit and activity data generated by Azure. In some cases, like the Resource Graph, this information is already generated and available directly for query. In other cases, like specific resource auditing, logs must be enabled by a resource owner. Generally, this uses Azure Monitor and is listed as the "Export Log" feature.
+
+### Understanding current inventory
+
+The Azure Resource Graph provides a Kusto-like query interface to all resources in a subscription. 
+
+Azure allows "Tags", which are simply key-value pairs, to be arbitrarily associated with almost any resource. Internally, Microsoft already uses this feature to track assets and configuration across subscriptions (e.g., AzSecPack, NRMSException). "Inventory" is an explicit use case supported in Azure.
+
+This effort will develop a set of standard Tags that will be applied to resources. These Tags will identify, at minimum, resources that are necessary for operation of our services.
```
Comment (MattGal) [inline Documentation/Project-Docs/vestigial-objects-onepager.md:46]: How do the tags get applied to the resources?

File: Documentation/Project-Docs/vestigial-objects-onepager.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  - [ ] Consider deployment information
+- [ ] Modify service deployments to publish inventory tags with deployed resources
+- [ ] Manually add inventory tags to resources not automatically deployed
+- [ ] Configure subscriptions to export to a designated Log Analytics Workspace
+  - [ ] Understand implications of Azure datacenter location and ingestion costs
+  - [ ] Evaluate the possibility of using deployments to enforce audit configuration
+  - [ ] Evaluate including Key Vault auditing information
+- [ ] Develop Dashboards presenting data to expected audience. Consider Grafana and Power BI (or a mix of both).
+  - [ ] FR can quickly view changes to an arbitrary resource
+  - [ ] New resources or resources with an unknown purpose are easily identifiable
```
Comment (MattGal) [inline Documentation/Project-Docs/vestigial-objects-onepager.md:75]: Maybe a nitpick but many of these tasks like "resources are easily identifiable" sound like goals.... consider separating them so we can think more easily about costing.

**Review by MattGal — COMMENTED** [review]
This looks great.  Could it get a bit more crisp on the tasks and maybe SWAG out their costs too?

**Review by MattGal — APPROVED** [review]

**Review by jonfortescue — APPROVED** [review]

**Pull request merged**